src/Runtime/PyExecutionSessionBase.cpp

```diff
@@ -105,7 +105,8 @@ std::vector<py::array> PyExecutionSessionBase::pyRun(
       dtype = ONNX_TYPE_INT32;
     else if (py::isinstance<py::array_t<std::int64_t>>(inputPyArray))
       dtype = ONNX_TYPE_INT64;
-    // string type missing
+    // else if (py::isinstance<py::array_t<const char *>>(inputPyArray))
```

Comment:
@chentong319 Here I added the missing string type which uses `const char *` pybind11 datatype.

Reply:
I have to comment this out for now until we get pybind11 support for it....I get the error `runtimeerror: numpy type info missing for pkc`.

Reply:
This is the first place we got assertion error. The link you provided is really helpful.